widget/embedded/src/constants/index.ts

```diff
@@ -1 +1,5 @@
+import { getExperimentalBaseUrl } from '../utils/quote';
+
 export const RANGO_PUBLIC_API_KEY = 'c6381a79-2817-4602-83bf-6a641a409e32';
+export const DEFAULT_BASE_URL = 'https://api.rango.exchange';
+export const RANGO_BASE_URL = getExperimentalBaseUrl() || DEFAULT_BASE_URL;
```

Comment:
It seems that changing the base URL has no effect. please check.

Reply:
It was my mistake. fixed.